python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```
@@ -1553,6 +1553,17 @@ $defs:
                     description: |
                       EOS cli commands rendered on router_bgp.vlans.
                       This configuration will not be applied to vlan aware bundles.
+              private_vlan:
+                type: dict
+                keys:
+                  type:
+                    type: str
+                    valid_values: ["community", "isolated"]
+                  primary_vlan:
+                    type: int
+                    convert_types:
+                      - str
+                    description: Primary VLAN ID.
```

Try this and rerun "pre-commit run schemas --all" to preview the markdown doc. It should not change. This way we only maintain it in one place and it is easier in the code with the classes (you will see once you write the code)

```suggestion
                $ref: eos_cli_config_gen#/keys/vlans/items/keys/private_vlan
```